infrastructure/aws/requirements-cdk.txt

```diff
@@ -7,4 +7,4 @@ boto3==1.28.71
 # pydantic settings
 pydantic~=2.0
 pydantic-settings~=2.0
-eoapi-cdk==6.0.0
\ No newline at end of file
+eoapi-cdk==6.1.2
```

Comment:
doesn't exist yet https://github.com/developmentseed/eoapi-cdk/pull/84

Reply:
should be 6.1.0 I think

Reply:
yep thanks, I changed this